Makefile

```diff
@@ -36,43 +36,59 @@ crosscompile: build/bin/$(BIN)-darwin-amd64 build/bin/$(BIN)-linux-amd64 build/b
 # platform-specific build
 build/bin/$(BIN)-darwin-amd64: $(SOURCE)
 	@mkdir -p build/bin
-	docker run --rm -v $(shell pwd):/go/src/github.com/$(ORGANISATION)/$(PROJECT) \
+	@mkdir -p go-build-cache
+	docker run --rm \
+		-v $(shell pwd):/go/src/github.com/$(ORGANISATION)/$(PROJECT) \
+		-v $(shell pwd)/go-build-cache:/.cache \
```

Comment:
I feel like it makes more sense to mount the local golang build cache

Reply:
That would only work for the architecture/os the developer is currently, on, but we are building for several architectures/os.

Reply:
I am fairly certain that the cache is smart enough to not confuse arches